reconf/test_stress_reconf.py

```diff
@@ -0,0 +1,124 @@
+"""
+On the fly reconfiguration stress test.
+"""
+
+__author__ = "Tempesta Technologies, Inc."
+__copyright__ = "Copyright (C) 2017-2024 Tempesta Technologies, Inc."
+__license__ = "GPL2"
+
+from helpers.tf_cfg import cfg
+from reconf.reconf_stress import LiveReconfStressTestBase
+
+SOCKET_START = f"{cfg.get('Tempesta', 'ip')}:443"
+SOCKET_AFTER_RELOAD = f"{cfg.get('Tempesta', 'ip')}:4433"
+STATUS_OK = "200"
+
+TEMPESTA_CONFIG = """
+listen ${tempesta_ip}:443 proto=h2;
+
+tls_certificate ${tempesta_workdir}/tempesta.crt;
+tls_certificate_key ${tempesta_workdir}/tempesta.key;
+tls_match_any_server_name;
+
+cache 0;
+server ${server_ip}:8000;
+"""
+
+
+class TestLiveReconf(LiveReconfStressTestBase):
```

Comment:
Please remove test `multiple_listeners.test_on_the_fly.TestOnTheFly`. It is duplicate.